README.md

```diff
@@ -0,0 +1,51 @@
+# Analog Forecast
+
+Make forecasts from similar meteorological conditions (analogs) using ERA5. 
+
+## Instructions
+
+1. Create the conda environment via `conda env create -f environment.yml`
+2. Activate the conda environment (`conda activate analog-forecast`)
+3. Store the path to the ERA5 daily data in the `DATA_DIR` environment variable: `export DATA_DIR=/atlas_scratch/kmredilla/analog_forecast/`
```

Comment:
Since there is no escape for the ```PROJECT_DIR``` variable in the ```config.py``` file, you also need to set the ```PROJECT_DIR``` environment variable before running the script from the interpreter